pages/src/footer/footer.less

```diff
@@ -46,38 +50,74 @@
     color: #0097E7;
   }
 
-  &-start-link:hover, &-link:hover,  &-dropdown-link:hover {
+  &-start-link:hover, &-link:hover,  &-accordion-link:hover {
     text-decoration: none;
     color: #00a6ff;
   }
 
-  &-link {
+  &-links {
     display: inline-block;
     font-size: 12px;
-    margin-right: 20px;
     margin-top: 20px;
   }
 
-  &-dropdown {
-    border-top: 1px solid white;
+  &-accordion-arrow::after {
+    content: '';
+
+    position: relative;
+    top: 8px;
+    left: 11px;
+    width: 7px;
+    height: 7px;
+
+    border-left: 2px solid white;
+    border-top: 2px solid white;
+
+    transform: rotate(225deg);
+    transform-origin: center;
+    transition: transform .5s ease-in-out, top .5s ease-in-out;
+  }
+
+  &-accordion-trigger {
+    &[active] {
+      .footer-accordion-arrow::after {
+        top: 12px;
+        transform: rotate(45deg);
+        transition: transform .5s ease-in-out, top .5s ease-in-out;
+        border-color: #00a6ff;
+      }
+    }
+  }
+
+  &-accordion {
     text-align: center;
     list-style-type: none;
     color: white;
+    border-top: 2px solid;
+    padding: 16px 0;
+    border-image: linear-gradient(to right, #131A2E, #00a6ff,#00a6ff, #ffffff, #131A2E) 47% 0%;
   }
 
-  &-dropdown-trigger {
+  &-accordion-trigger {
     display: flex;
     align-items: center;
     justify-content: center;
     height: 46px;
   }
 
-  &-dropdown-title {
-    font-weight: 500;
+  &-accordion-trigger[aria-expanded="true"]{
```

Comment:
Do not use aria attribute in the styles!!
Aria attributes only consumes the state of component but they should not deliver it